.github/containers/x86_64-broadwell-cuda11.6.1/repo/packages/AMS/package.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# AMSLib Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from spack.package import *
+
+
+class Ams(CMakePackage, CudaPackage):
+
+    homepage = "https://lc.llnl.gov/gitlab/autonomous-multiscale-project/marbl-matprops-miniapp"
```

Comment:
I believe here you pick our local AMS-package.py. There should be a public one in the spack official repo.

Reply:
We never do spack install AMS. Which we may need to start considering doing as a integration test. But let us do that in a next iteration.

Reply:
Makes sense to do it at a later iteration. I copied whatever we define for our CPU CI